Mochi Diffusion/Views/JobQueueView.swift

```diff
@@ -143,19 +143,19 @@ private struct InfoPopoverView: View {
 
             controller.currentModel = config.model
 
-            if let startingImage = config.pipelineConfig.startingImage {
+            if let startingImage = config.pipelineConfig.initImage, let strength = config.pipelineConfig.strength {
                 controller.startingImage = startingImage
-                controller.strength = Double(config.pipelineConfig.strength)
+                controller.strength = Double(config.pipelineConfig.strength!)
```

Comment:
We already have the strength, we could just use it.
```suggestion
                controller.strength = Double(strength)
```

Reply:
fix it